### ISSUE TITLE
running problem

### ISSUE THREAD

**Issue by wacharlin** [DmitryKey/bert-solr-search#8]
hi, thank you for your reply.i have solved the problem so i closed it. but now i run the code"streamlit run src/search_demo_solr.py " and i meet the new problem that i don't know how to do, there is a prolem that status is 500 for bert.but for bm25 i can run it.how to solve it?

![image](https://user-images.githubusercontent.com/38215077/112716206-f27d7b00-8f1f-11eb-9779-4026de348ccf.png)

**Comment by DmitryKey** [comment]
hi @wacharlin would you mind attaching a screenshot of the UI when this error happens?

**Comment by wacharlin** [comment]
> hi @wacharlin would you mind attaching a screenshot of the UI when this error happens?
this is the error.
![image](https://user-images.githubusercontent.com/38215077/112718816-5ad45880-8f30-11eb-9900-30b022cbee70.png)

**Comment by wacharlin** [comment]
> hi @wacharlin would you mind attaching a screenshot of the UI when this error happens?

i print the resp.json() (in solr_client.py) and find the error.

![image](https://user-images.githubusercontent.com/38215077/112719396-c5d35e80-8f33-11eb-8286-20d3e60ec997.png)

**Comment by wacharlin** [comment]
Hi,maybe I found the cause of this error, but I don't know how to solve it.
In my solr,I don't have "vector".I took a screenshot and compared it with yours. maybe the error is here.
I just run the code" src/index_dbpedia_abstracts_solr.py".because there is no file named "index_dbpedia_abstracts.py".
In "src/index_dbpedia_abstracts_solr.py",You originally gave 5 parameters, there are bc, but the parse_dbpedia_data() function only accepts two parameters, so I only kept the two parameters of source_file and max_docs, so I did not pass bc, and in the body of the function, I did not find Operations related to bert. So I think the problem may lie here. How can I solve it?
thank you.
![image](https://user-images.githubusercontent.com/38215077/112744627-4fd30400-8fd4-11eb-9c90-09284f40debb.png)

![image](https://user-images.githubusercontent.com/38215077/112744610-329e3580-8fd4-11eb-94b1-53c1d1d4c4f4.png)
